oauth2_provider/views/base.py

```diff
@@ -132,6 +133,8 @@ def form_valid(self, form):
     def get(self, request, *args, **kwargs):
         try:
             scopes, credentials = self.validate_authorization_request(request)
+            credentials["code_challenge"] = request.GET.get("code_challenge", None)
+            credentials["code_challenge_method"] = request.GET.get("code_challenge_method", None)
```

Comment:
Have you tried to not specify these two lines ? These seems unused until overriden lines 155/156.